templates/zerver/help/include/import-get-your-organization-started.md

```diff
@@ -0,0 +1,17 @@
+Once the import process is completed, you will need to:
+
+{start_tabs}
+
+1. [Review the settings for your new Zulip organization](/help/customize-organization-settings).
+
```

Comment:
I don't want to make this change; we are intentionally hinting at some settings folks are likely to want to review.

Would it work to just change "exported from Slack" -> "exported" in the current wording?

Reply:
This info is repeated in https://zulip.com/help/import-from-slack#import-details so I figured we could leave it out, but maybe it's best to reiterate here and remove "from Slack".

Reply:
Yeah, I see the reasoning, but I think it's good to emphasize it an extra time.